app/views/catalog/_homepage_sinai.html.erb

```diff
@@ -31,32 +31,32 @@
       </div><!--Grid row1 - Large image-->
 
       <!--Grid row2 - Medium images-->
-      <div class="row mx-0 pt-5">
+      <!-- <div class="row mx-0 pt-5">
         <div class="col-md-3 img-grid-sm img-grid-sm-lf"></div>
           <div class="
-            text-grid 
-            col-md-3 
-            d-flex 
-            flex-column 
-            justify-content-center 
-            align-items-start 
+            text-grid
+            col-md-3
+            d-flex
+            flex-column
+            justify-content-center
+            align-items-start
             pt-4 pt-md-0 pl-0 pl-md-4">
             <h3 class="grid-subtitle-sm">Bibliography</h3>
             <p class="grid-text">A list of catalogues and resources related to the manuscripts of St. Catherine’s Monastery.</p>
           </div>
         <div class="col-md-3 img-grid-sm img-grid-sm-rt mt-4 mt-md-0"></div>
         <div class="
-          text-grid 
-          col-md-3 
-          d-flex 
-          flex-column 
-          justify-content-center 
+          text-grid
+          col-md-3
+          d-flex
+          flex-column
+          justify-content-center
           align-items-start 
           pt-4 pt-md-0 pl-0 pl-md-4">
           <h3 class="grid-subtitle-sm">About the Collection</h3>
           <p class="grid-text">Currently, the digital collection comprises 75 Arabic and Syriac manuscripts dating from the 7th to the 17th centuries. New manuscripts will be added regularly.</p>
       </div>
-     </div><!--Grid row2 - Medium images-->
+     </div><!--Grid row2 - Medium images--> -->
```

Comment:
@kirschbombe On line 59, there are two closing comment tags (-->) following each other. The last tag will probably render as plain text and show up on screen. To fix, place the last closing tag right after the closing div. 

`</div><!--Grid row2 - Medium images--> -->`
should be
`</div>--> <!--Grid row2 - Medium images-->`

Reply:
Fixed it :)